src/transforms/transform-pull-request.ts

```diff
@@ -157,3 +159,84 @@ const getBranches = async (gitHubInstallationClient: GitHubInstallationClient, p
 		}
 	];
 };
+
+export const transformPullRequest = (_jiraHost: string, pullRequest: pullRequestNode, reviews?: pullRequestNode["reviews"], log?: Logger) => {
+	const issueKeys = extractIssueKeysFromPr(pullRequest);
+
+	if (isEmpty(issueKeys) || !pullRequest.headRef.repository) {
+		log?.info({
+			pullRequestNumber: pullRequest.number,
+			pullRequestId: pullRequest.id
+		}, "Ignoring pullrequest since it has no issue keys or repo");
+		return undefined;
+	}
+
+	try {
+		return {
+			author: getJiraAuthor(pullRequest.author),
+			commentCount: pullRequest.comments.totalCount || 0,
+			destinationBranch: pullRequest.baseRef?.name || "",
+			destinationBranchUrl: `https://github.com/${pullRequest.baseRef?.repository?.owner?.login}/${pullRequest.baseRef?.repository?.name}/tree/${pullRequest.baseRef?.name}`,
+			displayId: `#${pullRequest.number}`,
+			id: pullRequest.number,
+			issueKeys,
+			lastUpdate: pullRequest.updatedAt,
+			reviewers: mapReviews(reviews?.nodes),
+			sourceBranch: pullRequest.headRef?.name || "",
+			sourceBranchUrl: `https://github.com/${pullRequest.headRef?.repository?.owner?.login}/${pullRequest.headRef?.repository?.name}/tree/${pullRequest.headRef?.name}`,
+			status: pullRequest.state, // test closed and declined behaviou// mapStatus(pullRequest.state, pullRequest.merged_at), mapStatus(pullRequest.state, pullRequest.mergedAt),
+			timestamp: pullRequest.updatedAt,
+			title: pullRequest.title,
+			url: pullRequest.url,
+			updateSequenceId: Date.now()
+		};
+	} catch (err) {
+		throw new Error();
+	}
+};
+
+const mapReviews = (reviews: pullRequestNode["reviews"]["nodes"] = []): JiraReview[] => {
```

Comment:
This is mostly a like for like copy of the existing mapReview, except its **no longer async** and has a few different attributes it targets

Reply:
Does it account for recent changes/fixes that @rachellerathbone is doing?

Reply:
And same: requsted_reviews are not accounted for

Reply:
requested reviews! Thank you!

We will have some gnarly merge conflicts but the mapReviews logic is pretty similar so will just need to pair with @rachellerathbone to make sure we get it right

Reply:
But my stuff is merged...

Reply:
No merge conflicts coz my changes have already been merged.